api/src/utils/business-logic-manager.js

```diff
@@ -124,7 +125,8 @@ function isIssuedToConsideredAnonymous(issuedTo, issuingAgency) {
   }
 
   // check if the issuingAgency has legislative authority to publish names
-  if (issuingAgency && !constants.AUTHORIZED_PUBLISH_AGENCIES.includes(issuingAgency)) {
+  if (issuingAgency && !constants.AUTHORIZED_PUBLISH_AGENCIES.includes(issuingAgency) 
```

Comment:
This is getting to be a lot to parse. We have a comment which is goo, we could also bind this to a constant:

`
hasLegislativeAuthorityToPublish = (
issuingAgency && 
!constants.AUTHORIZED_PUBLISH_AGENCIES.includes(issuingAgency) && !constants.AUTHORIZED_PUBLISH_AGENCIES.includes(agenciesController.getAgencyCodeFromNameBandaid(issuingAgency))
)
`